sketcherMinimizer.cpp

```diff
@@ -270,19 +272,19 @@ void sketcherMinimizer::initialize(
 
     flagCrossAtoms();
 
-    m_minimizer._atoms = _atoms;
-    m_minimizer._bonds = _bonds;
-    m_minimizer._molecules = _molecules;
-    m_minimizer._residues = _residues;
-    m_minimizer._residueInteractions = _residueInteractions;
+    m_minimizer._atoms = m_atoms;
+    m_minimizer._bonds = m_bonds;
+    m_minimizer._molecules = m_molecules;
+    m_minimizer._residues = m_residues;
+    m_minimizer._residueInteractions = m_residueInteractions;
```

Comment:
These need the `m_` prefix on both sides of the assignment operator. e.g.:

    m_minimizer.m_atoms = m_atoms;

Reply:
Ah I didn't add the `m_` prefix to CoordgenMinimizer fields, but I can update that.